### ISSUE TITLE
Can't receive metadata with tags 

### ISSUE THREAD

**Issue by coherent-edkarhuber** [contentful/contentful.net#280]
Hello,
I use Content Management API (via IContentfulManagementClient) to get entry tags. For some reason, Metadata property is always `null`
Code example: `var entry = await _client.GetEntry(entryId);` which returns `Entry<dynamic>`.
I set the tag via UI, but again `GetEntry` method returns me entry with all data except Metadata tags.
Could you point me how I can receive tags for a specific entry?
Thanks.

Update:
I tried to check if getting collection entries returns me metadata, so yes, it works:
```csharp
var builder = QueryBuilder<Entry<dynamic>>.New.ContentTypeIs(Constants.ContentType.Application)
                .FieldEquals(x => x.SystemProperties.Id, entryId);

var entries = await _client.GetEntriesCollection(builder);
```
In this case entry has Metadata which is not null...

**Comment by Roblinde** [comment]
Hi @coherent-edkarhuber thanks for the report. This looks like a potential bug. The metadata doesn't seem to be added for the single entry endpoint. I'll look into getting a fix into the next version.

**Comment by coherent-edkarhuber** [comment]
thanks

**Comment by bacpac** [comment]
For anyone having a similar problem with not getting metadata in the response; after digging into this git repo I found the metadata JSON keys are renamed from `metadata` to `$metadata` in the SDK. Not sure why, but perhaps to avoid certain conflicts.

Adding `[JsonProperty("$metadata")]` to my Gateway model mapped the metadata correctly.

**Comment by ErikEJ** [comment]
Yes  it look like there is a gap in the documentation.

**Comment by Roblinde** [comment]
Apologize for the delay, this is now fixed in version 7.1.0
